releases/release-1.22/OWNERS

```diff
@@ -1,17 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
```

Comment:
I don't think this file should be deleted?

Reply:
Good catch. Looking into it

Reply:
Resolved it - https://github.com/savitharaghunathan/sig-release/tree/bootstrap-release-1.22/releases/release-1.22